test/spec/Feature/Query/RangeSpec.hs

```diff
@@ -126,7 +126,8 @@ spec = do
                 "hint":null
               }|]
             { matchStatus  = 416
-            , matchHeaders = ["Content-Range" <:> "*/0"]
+            , matchHeaders = [ "Content-Range" <:> "*/0"
+                             , "Content-Length" <:> "144"]
```

Comment:
The test only responds with the `Content-Length` when it's added to `matchHeaders`... this is unexpected for a test since it shouldn't modify how our application works. Unless there's a way to make this same request and avoid getting the "Content-Length" or ignore it?